Tg frontend

NewalexOA opened the pull request:
add 'report' & import from actions.py

NewalexOA committed:
add keyboard and comments

NewalexOA committed:
update menu buttons

NewalexOA committed:
update code in main.py

add 'report' & import from actions.py

serge-auro commented on main.py:
```diff
@@ -1,17 +1,112 @@
+    'status': 'Статус',
+    'report': 'Отчеты',
+    'edit': 'Редактировать',
+    'add': 'Добавить',
```
в дизайне нет кнопки /add, есть только /new_habit

serge-auro commented on main.py:
```diff
@@ -1,17 +1,112 @@
+    'menu': 'В главное Меню',
+    'status': 'Статус',
+    'report': 'Отчеты',
+    'edit': 'Редактировать',
```
в дизайне нет кнопки /edit есть только /edit_habit

serge-auro commented on main.py:
```diff
@@ -1,17 +1,112 @@
+    'report': 'Отчеты',
+    'edit': 'Редактировать',
+    'add': 'Добавить',
+    'delete': 'Удалить',
```
в дизайне нет кнопки /delete, есть только /del_habit

serge-auro commented on main.py:
```diff
@@ -1,17 +1,112 @@
+    'edit': 'Редактировать',
+    'add': 'Добавить',
+    'delete': 'Удалить',
+    'done': 'Отметить V',
```
в дизайне нет кнопки /done, есть только /mark_habit

serge-auro reviewed (commented):
надо сделать рефакторинг - везде переименовать кнопки в соответствии с дизайном проекта:
add -> new_habit
edit -> edit_habit
delete -> del_habit
done -> mark_habit

NewalexOA committed:
update main

поменял названия в соответствии с дизайном и пересмотрел концепцию - сделал инлайн-кнопки

serge-auro committed:
Merge branch 'develop' into TG-frontend
